mapstruct-lombok/pom.xml

```diff
@@ -36,22 +36,18 @@
     </properties>
 
     <dependencies>
-        <dependency>
-            <groupId>org.projectlombok</groupId>
-            <artifactId>lombok</artifactId>
-            <version>${org.projectlombok.version}</version>
-            <scope>provided</scope>
-        </dependency>
 
         <dependency>
             <groupId>org.mapstruct</groupId>
             <artifactId>mapstruct</artifactId>
             <version>${org.mapstruct.version}</version>
         </dependency>
+
+        <!-- lombok dependencies should not end up on classpath -->
         <dependency>
-            <groupId>org.mapstruct</groupId>
```

Comment:
If you remove the mapstruct processor then stuff won't work in IntelliJ pre 2018.1.1

Reply:
@filiphr  Is this still an issue? Should we leave it as (than I'll close this one)?

Reply:
I think we are fine now. IntelliJ is releasing 2019.1 in few weeks and I think people update regularly.